src/cashweb/wallet/index.ts

```diff
@@ -272,7 +271,7 @@ export class Wallet {
     const scriptHashes = Object.keys(this.electrumScriptHashes)
```

Comment:
Could use your refactor of addresses and changeAddresses here instead.  Though I suppose you're using electrum for notifications and a few things still.

Reply:
Yeah, one thing at a time. There's not tests, so code updates are like shooting in the dark.